### DISCUSSION TITLE
How to ignore file in live reload

### DISCUSSION THREAD

**Discussion by Abner-Corona** [quasarframework/quasar#12639] [CLI - SSR mode]
Hi, where are the configuration for ignore a file in ssr dev?
Thanks

**Comment by MilosPaunovic** [comment]
What do you mean by ignoring a file? The more details you provide to your question, the better (and quicker) answer community might provide.

**Comment by Abner-Corona** [comment]
I mean unwatch a file 😅, when edit a file not reload the project, i guess are in the webpack chain configuration
